roles/openshift_node_dnsmasq/tasks/network-manager.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Install network manager dispatch script
+  copy:
+    src: networkmanager/99-origin-dns.sh
+    dest: /etc/NetworkManager/dispatcher.d/
+    mode: 0755
+  register: dispatch_script
+  
+- name: Restart Network Manager
+  service: 
+    name: NetworkManager
+    state: restarted
+  when: dispatch_script.changed | bool
+
+- name: Test dns resolution
+  command: >
+    getent hosts google.com
```

Comment:
Yeah, I can't come up with a decent alternative however. If I were to check for kubernetes.default that'll fail if the node and SDN haven't been started. What do you think about removing it entirely? It was a nice safety net for me while developing but I'm not certain it's necessary long term once QE has finished testing.